internal/compiler/passes/embed_images.rs

```diff
@@ -364,13 +364,12 @@ fn load_image(
         || file.canon_path.extension() == Some(OsStr::new("svgz"))
     {
         let tree = i_slint_common::sharedfontdb::FONT_DB.with_borrow(|db| {
-            let mut options = usvg::Options::default();
-            options.fontdb = (*db).clone();
+            let option = usvg::Options { fontdb: (*db).clone(), ..Default::default() };
```

Comment:
Why to you rename to `option` (singular) despite the type is called `Options` (plural)